RiotSharp/Interfaces/IRiotApi.cs

```diff
@@ -148,35 +148,35 @@ public interface IRiotApi
         /// Retrieves the league entries for the specified summoners, you can submit more than 10 summoner ids.
         /// </summary>
         /// <param name="region">Region in which you wish to look for the leagues of summoners.</param>
-        /// <param name="summonerIds">The summoner ids, not limited to 10.</param>
+        /// <param name="summonerId">The summoner ids, not limited to 10.</param>
```

Comment:
Update parameter doc to "summoner id" (singular) here and elsewhere in the file.

Reply:
Arf and i have forgottent the other Summoners too, my bad.